xiaomi.vacuum.gen1/firmwarebuilder/imagebuilder.sh

```diff
@@ -15,20 +15,40 @@
 #You should have received a copy of the GNU General Public License
 #along with this program. If not, see <http://www.gnu.org/licenses/>.
 # Preparation:
-# place english.pkg and v11_003077.pkg in this folder
+# place english.pkg and v11_<fw_version>.pkg (e.g. v11_003077.pkg) in this folder
 # place your authorized_keys in this folder
 #
-if [ ! -f /usr/bin/ccrypt ]; then
+IS_MAC=false
+FIRMWARE_VERSION=$1
+
+if [[ ! -n "$FIRMWARE_VERSION" ]]; then
```

Comment:
I almost forgot this! I found it cumbersome to change the firmware version manually so I made it a parameter of the script. If applicable I would add support for changing firmware versions to other scripts as well.